compiler/test/stdlib/immutablepriorityqueue.test.gr

```diff
@@ -4,8 +4,6 @@ include "immutablepriorityqueue"
 include "list"
 include "array"
 
-assert ImmutablePriorityQueue.make(compare) == ImmutablePriorityQueue.empty
-
```

Comment:
`compare` in one module is no longer necessarily equal to `compare` in another module because they can be different closures. There's no real equivalent of this test now. `==` currently just compares pointers for closures. 

A test that checks something similar to this would be this: `assert ImmutablePriorityQueue.isEmpty(ImmutablePriorityQueue.make(compare))`, but I do not believe that was the essence of this test.